ci/deps/actions-38-minimum_versions.yaml

```diff
@@ -32,7 +32,7 @@ dependencies:
   - gcsfs=2021.07.0
   - jinja2=3.0.0
   - lxml=4.6.3
-  - matplotlib=3.3.2
+  - matplotlib=3.6.0
```

Comment:
Why did matplotlib need to be bumped? 3.6 was only released in September 2022 so it's fairly new

Reply:
3.6.0 is the first version that doesnt do `series[:, None]`

Reply:
So our existing plotting tests were failing before this bump?

Could you also update this version in `install.rst`?

Reply:
yes and yes

Reply:
Cool, I think some matplotlib code can be cleaned up with this bump if interested in following up

Reply:
sure.  something to grep for to find the relevant code?

Reply:
Originally in `pandas/plotting/_matplotlib/compat.py`, I think anything that uses `mpl_ge_3_4_0`  or `mpl_ge_3_5_0`

Reply:
The min version build is failing I think with this bump, so the cleanup would need to be done sooner rather than later: https://github.com/pandas-dev/pandas/actions/runs/3380325519/jobs/5612984854

Reply:
ive put it into a CLN branch on which the tests are about to finish locally, will push shortly